tsconfig.json

```diff
@@ -8,13 +8,10 @@
 		"resolveJsonModule": true,
 		"skipLibCheck": true,
 		"sourceMap": true,
-		"strict": true,
-		"noImplicitAny": true,
```

Comment:
Why did you remove this strict options?

Reply:
I synced some of the configs with the create svelte package.

In this case: https://github.com/sveltejs/kit/blob/master/packages/create-svelte/shared/%2Btypescript/tsconfig.json

Reply:
The options you have removed make TypeScript less strict

Reply:
Ok, I added them back :)